openshift/tests-extension/README.md

```diff
@@ -26,6 +26,32 @@ Use the environment variable `KUBECONFIG` to point to your cluster configuration
 KUBECONFIG=path/to/kubeconfig ./bin/olmv1-tests-ext run-test -n <test-name>
 ```
 
+### Local Test using OLMv1 and OCP 
+
+1. Use the cluster-bot to create an OpenShift cluster with OLMv1 installed.
+
+**Example:**
+
+```shell
+launch 4.20 gcp,techpreview
+```
+
+2. Set the `KUBECONFIG` environment variable to point to your OpenShift cluster configuration file.
+
+**Example:**
+
+```shell
+mv ~/Downloads/cluster-bot-2025-08-06-082741.kubeconfig ~/.kube/cluster-bot.kubeconfig
+export KUBECONFIG=~/.kube/cluster-bot.kubeconfig
+```
+
+3. Run the tests using the `olmv1-tests-ext` binary.
+
+**Example:**
+```shell
+./bin/olmv1-tests-ext run-suite olmv1/all
+```
+
```

Comment:
Can this whole section be condensed to: 


>An Openshift Cluster with olmv1 enabled is required to run the tests. Once the `KUBECONFIG` env variable is pointing to the kubeconfig file saved locally (eg `~/.kube/cluster-bot.kubeconfig`), the tests can be run using the `olmv1-tests-ext` binary: 
```shell
./bin/olmv1-tests-ext run-suite olmv1/all
```

That way we're not making assumptions about how the reader is getting their OCP clusters (there are other ways than using cluster-bot), or getting into "how do you get a cluster" considerations at all in this section of the doc.

Reply:
Hi @anik120,

Thanks for reviewing this. I think it's important to keep the detailed, step-by-step examples in this section. The goal is to make it as simple as possible for anyone to run the tests, regardless of their background, and the code examples help with that a lot.

I also believe we need to be clear about the `techpreview` requirement. The current examples help to clarify this.

Would you be open to keeping the more detailed version? 
This way, we can make sure the documentation is very clear and easy to follow.

Also, it might make more sense for you if you open the doc in the review tab and preview it.
You will see that we have generic info above, and we add examples with both. (OCP and k8s )